pyproject.toml

```diff
@@ -24,7 +24,7 @@ DJANGO_SETTINGS_MODULE = "tin.settings"
 python_files = "tests.py test_*.py *_tests.py"
 norecursedirs = ["media", "migrations", "sandboxing"]
 testpaths = "tin"
-addopts = "--doctest-modules tin --import-mode=importlib"
+addopts = "--doctest-modules tin --import-mode=importlib -n 8"
```

Comment:
Do you have a reason for choosing 8?

Reply:
It's the amount of workers Manim uses on my computer while building :)
Do you want me to lower it?

Reply:
No it's fine - just curious if you had a specific reason